### ISSUE TITLE
How to get figures in the margin?

### ISSUE THREAD

**Issue by fkgruber** [ox-tufte/ox-tufte#10]
Hi 
What is the proper way to get figures in the margin? Can we have captions as well?
thanks

**Comment by dakrone** [comment]
@fkgruber thanks for opening this, I haven't tested in a while, what do figures in the margin notes currently do?

And can you expand on what you'd like captions to do? Just look like the tufte captions if they're in `#+CAPTION: foo bar baz`?

**Comment by fkgruber** [comment]
Hi.
Just wanted to do something like:
https://rstudio.github.io/tufte/

**Comment by bayesians-inc-PL** [comment]
this is now possible in the upcoming v2 release. while images can be embedded in either the `call_marginnote` (inline syntax) or `#+begin_marginnote` (block syntax) syntax, if we also want captions, then you have to use the block syntax.
```org
#+begin_marginnote
an image with caption:
#+CAPTION: caption for image in marginnote
[[../../../static/authors/Chris.png]]
#+end_marginnote
```

the v2 release is now available for preview at https://github.com/ox-tufte/ox-tufte/tree/dev .
for everyone's awareness, https://github.com/melpa/melpa/pull/8633 needs to be merged before `dev` branch can be merged into `main`
